Admin/View/AutomationViewBuilder.php

```diff
@@ -44,6 +47,18 @@ public function setEntityClass(string $entityClass): AutomationViewBuilderInterf
         $this->addRequestParameter('entityClass', $entityClass);
         $this->addMetadataRequestParameters(['entityClass' => $entityClass]);
 
+        $this
```

Comment:
guess we need a something like `method_exists` here, not sure how we did this in other bundles

Reply:
No, because I increased the min requirement of sulu to `^2.1` and the corresponding sulu pr for badges will be merged into `2.1` as well

Reply:
but `^2.1` will also match `2.1.7` which does not contain the class 🙂  we can also use `^2.1.8` once it is released

Reply:
🤦 okay, would use `^2.1.8` then

Reply:
2.1.8 will also match 2.2.0 so I would really add here the method_exists check and just go with the sulu version before.

Reply:
I think it wouldn't hurt too much to increase the min-requirement of `sulu/sulu` to kind of force people to update, if they want to use the automation bundle. WDYT? And it's really no big deal to update from `2.0` to `2.1` or even `2.2`

Reply:
But I'm also fine with adding the `method_exists()` check